string mission_name
string goal_name
---
